Containers/mastercontainer/start.sh

```diff
@@ -277,4 +277,7 @@ https://your-domain-that-points-to-this-server.tld:8443"
 # Set the timezone to UTC
 export TZ=UTC
```

Comment:
Why is the timezone forced to be utc?

Reply:
Because the logic in the mastercontainer assumes that the timezone is UTC.

Reply:
However of course all other containers except the borgbackup container use the timezone that is chosen in the AIO interface.

Reply:
> Because the logic in the mastercontainer assumes that the timezone is UTC.

so it breaks when using another timezone?

Reply:
Yes, basically